agent_code/auto_bomber/model.py

```diff
@@ -9,25 +10,42 @@
 
 
 class LinearAutoBomberModel:
-    def __init__(self, feature_extractor):
+    def __init__(self, train, feature_extractor):
+        self.train = train
         self.weights = None
         self.feature_extractor = feature_extractor
-        self.writer = SummaryWriter()
 
-        path = Path(config.MODEL_PATH)
-        if path.is_file():
-            with path.open(mode="rb") as file:
+        if config.MODEL_DIR and Path(config.MODEL_DIR).is_dir():
+            self.model_dir = Path(config.MODEL_DIR)
+        elif config.MODEL_DIR and not Path(config.MODEL_DIR).is_dir():
+            raise FileNotFoundError("The specified model directory does not exist!\nIf you wish to train a NEW model"
+                                    "set parameter to None, otherwise specify a valid model directory.")
+        else:
+            model_index = sorted([int(x.stem) for x in Path(config.MODELS_ROOT).iterdir() if x.is_dir()])[-1]
```

Comment:
this will fail on a empty directory

Reply:
will still fail due to index out of bound - I fixed it in my version? Shall I simply put my changes in here?